frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js

```diff
@@ -117,17 +118,41 @@ class AddNewMovieSearchResult extends Component {
                   />
               }
 
-              <Link
-                className={styles.tmdbLink}
-                to={`https://www.themoviedb.org/movie/${tmdbId}`}
-                onPress={this.onTMDBLinkPress}
-              >
-                <Icon
-                  className={styles.tmdbLinkIcon}
-                  name={icons.EXTERNAL_LINK}
-                  size={28}
-                />
-              </Link>
+              {
+                isSmallScreen ?
```

Comment:
Shouldn't be needed now.

Reply:
Needed to move the row of buttons down for mobile devices